api/openapi/api.yaml

```diff
@@ -0,0 +1,74 @@
+openapi: 3.0.0
+info:
+  title: Schematic REST API
+  version: 0.1.0
+  description: >-
+    This service exposes core functionalities from schematic as REST API
+    endpoints
+
+servers:
+  - url: /v1
+
+paths:
+  /manifest/generate:
+    get:
```

Comment:
Generally, I like to use the `POST` verb here for creating resources on the backend.

Reply:
Quick question here. Are you suggesting the use of `POST` over `GET` for security reasons? In that, if the request is `GET` users could easily create resources by hitting endpoints from the browser? I think `POST` has some amount of that risk too?

Reply:
Yes, there are some other advantages to using `POST` vs `GET`. `GET` requests can be cached by a browser and  ultimately the same request can be executed again with or without a user being present. Perhaps a browser plugin vulnerability.

I googled it, but this table sums it up https://www.diffen.com/difference/GET-vs-POST-HTTP-Requests

Reply:
Thank you for the explanation Jason, this is helpful. For the purposes of this endpoint, which has to do with creation of the manifest, I think it should be fine. The next endpoints to come, validation and submission will be `POST`.